homeassistant/components/moonraker/binary_sensor.py

```diff
@@ -0,0 +1,101 @@
+"""Binary sensors for Moonraker API integration."""
```

Comment:
There are multiple platforms in this pull request, please make your PR as small as possible.

See: <https://developers.home-assistant.io/docs/creating_component_code_review/#5-make-your-pull-request-as-small-as-possible>

Reply:
@frenck The issue is that the integration is used for monitoring 3d printing, as such it's not really possible to deliver only a single platform and the integration be considered useful. The sensors included represent a minimal subset that are common in monitoring setups.

Reply:
You can add other platforms in follow-up pull requests.
Please limit this PR.

Reply:
+1 on reducing the size of the PR. The initial PR can take a while, but once it is merged subsequent PRs are usually much quicker.

Reply:
It's not a matter of opinion or +1's or something. It is part of our contribution guidelines and requirements.

Reply:
I removed button and binary_sensor for now.